extensions/binding/org.eclipse.smarthome.binding.astro/src/main/java/org/eclipse/smarthome/binding/astro/internal/job/Job.java

```diff
@@ -39,7 +40,7 @@
 
     /** Logger Instance */
     public final Logger logger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
-
+    
```

Comment:
These blanks just slipped in. 
Are you using the ESH IDE with the preset formatter? (Just asking to see whether it has a bug we need to fix...)